README.md

```diff
@@ -8,17 +8,7 @@ A bundle that extend the Pim Enterprise CatalogRuleBundle, adding the possibilit
 
 ## Installation
 
-As the Git repository is private, you need to add it to the composer.json of your project:
-
-    "repositories": [
-            {
-                "type": "vcs",
-                "url": "https://github.com/akeneo-labs/ClassificationRuleBundle.git",
-                "branch": "master"
-            }
-        ]
-
-Then install the bundle with composer:
+You can install the bundle with composer:
 
     $ php composer.phar require akeneo-labs/classification-rule-bundle:1.0.*
```

Comment:
Btw, this part is not in my PR but I agree we have to change that documentation.